components/News/NewsItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FunctionComponent } from "react";
+import Link from "next/link";
+import { TypeNews } from "../../generated/contentful";
+import Image from "next/image";
+import Style from "./styles.module.scss";
+import clsx from "clsx";
+
+export interface NewsItemProps {
+  newsItem: TypeNews;
+  className?: string;
```

Comment:
You should be able to get this for free by using the right base for props: `React.FunctionComponent<
  Props & React.HTMLAttributes<HTMLDivElement>
>`